### PR TITLE
enable data export from admin panel

### PR DESCRIPTION
**Pull request by robguthrie**
This adds a "toggle data export" button to the groups show page.

It also adds a features object much like the experiences object, for storing which features are enabled and any details we need about them.

### CODE REVIEW

**Comment by robguthrie** [comment]
I've tested this. I don't think there is much to worry about.. @gdpelican can you tell me if this needs more QA or anything?

**Comment by gdpelican** [comment]
Looks great to me. 

Do you think we could (would it be smart to, not should we do it right now) move some other flags, like whether a group is experimental or not, into features?

**Comment by robguthrie** [comment]
yes, I do think we should do that. it's a nice way to keep plugin specific
stuff out of the main schema.

**Comment by robguthrie** [comment]
deploying to clone now.

**Comment by gdpelican** [comment]
What's the difference between 'features' and 'experiences' on groups?

**Comment by robguthrie** [comment]
In my mind, features are things that the group can do or use. Like, which comment form widget they are using. Or if they have data export.

Experiences is where we record that the group has done something, like seen a modal, or pushed a button.

**Comment by robguthrie** [comment]
Would could mix them up.. but I don't think that would be helpful in the long run.

**Comment by gdpelican** [comment]
Sounds good to me

File: app/models/ability.rb [hunk 1]
```diff
@@ -66,8 +66,7 @@ def initialize(user)
     end
 
     can :export, Group do |group|
-      user.is_admin? or
```
Comment (mhjb) [inline app/models/ability.rb]: @robguthrie how come you took this out?

Comment (robguthrie) [inline app/models/ability.rb]: i guess it was because it's an undocumented thing.

Happy to put it back in if you like. Sorry for the inconvenience.

Comment (mhjb) [inline app/models/ability.rb]: I understand user.is_admin = true means user has access to loomio.org/admin, eh?

Comment (robguthrie) [inline app/models/ability.rb]: yes

**Pull request merged**